content/en/blog/_posts/2024-07-31-consistent-read-from-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog
+title: 'Kubernetes Consistent Reads from Cache: A Leap in Performance and Scalability'
```

Comment:
I'd expect the title to match the other post-release articles with a particular prefix.

Reply:
Done